packages/venia-concept/src/components/MiniCart/__tests__/cartOptions.spec.js

```diff
@@ -50,40 +44,6 @@ const configItem = {
     ]
 };
 
-test('renders with options when passed a configurable item', () => {
-    const updateCart = jest.fn();
-    const closeOptionsDrawer = jest.fn();
-    const wrapper = shallow(
-        <CartOptions
-            cartItem={cartItem}
-            configItem={configItem}
-            updateCart={updateCart}
-            closeOptionsDrawer={closeOptionsDrawer}
-            classes={classes}
-        />
-    ).dive();
-
-    const options = wrapper.find('section.options');
-    expect(options.length).toBe(1);
-});
-
-test('renders with an empty configurable item, does not display options', () => {
-    const updateCart = jest.fn();
-    const closeOptionsDrawer = jest.fn();
-    const wrapper = shallow(
-        <CartOptions
-            cartItem={cartItem}
-            configItem={{}}
-            updateCart={updateCart}
-            closeOptionsDrawer={closeOptionsDrawer}
-            classes={classes}
-        />
-    ).dive();
-    const options = wrapper.find('section.options');
-    expect(options.length).toBe(0);
-    expect(wrapper.hasClass(classes.root)).toBe(true);
-});
```

Comment:
These tests are no longer needed because they test the internals of a different component (`options.js`), not anything in `cartOptions.js`.

Reply:
I love it! Thanks for maintaining and streamlining the test suite, and for keeping it min mind not to test implementation details.